apps/sms/services/test/unit/drafts_test.js

```diff
@@ -386,6 +386,28 @@ suite('Drafts', function() {
     });
   });
 
+  suite('getAllThreadless()>', function() {
+    setup(function() {
+      [
+        threadDraft1,
+        draft1,
+        draft2
+      ].forEach(Drafts.add, Drafts);
+    });
+
+    teardown(function() {
+      Drafts.clear();
+    });
+
+    test('etAllThreadless returns only thread less drafts', function() {
+      var threadlessDrafts = Drafts.getAllThreadless();
+
+      assert.equal(threadlessDrafts.length, 2);
+      assert.deepEqual(threadlessDrafts[0], draft1);
+      assert.deepEqual(threadlessDrafts[1], draft2);
```

Comment:
how about replacing the 3 previous lines with `assert.deepEqual(threadlessDrafts, [ draft1, draft2 })` ?